Octokit/Clients/RepositoriesClient.cs

```diff
@@ -502,7 +521,8 @@ public Task<Branch> GetBranch(string owner, string repositoryName, string branch
             Ensure.ArgumentNotNullOrEmptyString(repositoryName, "repositoryName");
             Ensure.ArgumentNotNullOrEmptyString(branchName, "branchName");
 
-            return ApiConnection.Get<Branch>(ApiUrls.RepoBranch(owner, repositoryName, branchName));
+            const string previewAcceptsHeader = "application/vnd.github.loki-preview+json";
```

Comment:
The licenses API? Yeah, there's two usages in `MiscellaneousClient` which could use a bit of :lipstick: to tidy up at some stage.

Let's define a field on the test class which gets reused here, and then look at the other usages separately.